adapters/bmtm/brightmountainmedia_test.go

```diff
@@ -1,19 +1,21 @@
 package bmtm
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/prebid/prebid-server/adapters/adapterstest"
 	"github.com/prebid/prebid-server/config"
 	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestJsonSamples(t *testing.T) {
 	bidder, buildErr := Builder(openrtb_ext.BidderBmtm, config.Adapter{
 		Endpoint: "https://example.com/api/pbs"})
 
 	if buildErr != nil {
```

Comment:
With `assert.NoError` you don't need the `if` condition here anymore. `assert.NoError` will make sure that `buildErr` is `nil` and if it isn't then fail the test with the error message you're passing to it as the last param